cmd/ctr/commands/images/import.go

```diff
@@ -119,6 +123,10 @@ If foobar.tar contains an OCI ref named "latest" and anonymous ref "sha256:deadb
 			return closeErr
 		}
 
+		if context.Bool("no-unpack") {
```

Comment:
LGTM, but I prefer `if !context.Bool("no-unpack"){ /* unpack */ }`

Reply:
SGTM, I'll update the PR in a moment! 👍

Reply:
done